QgisModelBaker/qgismodelbaker.py

```diff
@@ -412,6 +436,41 @@ def show_about_dialog(self):
         self.msg.setStandardButtons(QMessageBox.Close)
         self.msg.exec_()
 
+    def init_validate_dock(self):
+        settings = QSettings()
+        self.__validate_dock = ValidateDock(self.ili2db_configuration, self.iface)
+        self.iface.addDockWidget(
+            settings.value(
+                "QgisModelBaker/validateDockArea", Qt.RightDockWidgetArea, type=int
+            ),
+            self.__validate_dock,
+        )
+        self.__validate_dock.visibilityChanged.connect(
+            self.__validate_action.setChecked
+        )
+        self.__validate_dock.setVisible(
+            settings.value("QgisModelBaker/dockWidgetIsVisible", False, type=bool)
```

Comment:
All validate dock widget settings could be in the same group. This one in particular would be misleading if in future we get a separate dock widget for another functionality.

Reply:
Good point. Will consider that.

Reply:
Will name it to `ValidationDockIsVisible`

Reply:
Or:
- `ValidationDock/IsVisible`
- `ValidationDock/Area`
So they are grouped in the settings viewer

Reply:
Right.